ouroboros-consensus/src/Ouroboros/Consensus/MiniProtocol/ChainSync/Client.hs

```diff
@@ -565,7 +565,7 @@ chainSyncClient mkPipelineDecision0 tracer cfg
           -- ('rollBackward'), so we have nothing to do.
           return $ Just kis
 
-        | Just intersection <- AF.intersectionPoint ourFrag' theirFrag ->
+        | Just (ourPre, _ourSuf, _theirPre, theirSuf) <- AF.intersect ourFrag' theirFrag ->
```

Comment:
Was just confused when looking at the docs; makes no semantic difference of course:
```suggestion
        | Just (ourPre, _theirPre, _ourSuf, theirSuf) <- AF.intersect ourFrag' theirFrag ->
```

Reply:
Fixed, thanks.